src/Build/Logging/BinaryLogger/Postprocessing/EmbeddedContentKind.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Build.Logging
+{
+    internal enum EmbeddedContentKind
+    {
+        Unknown = -1,
+        ProjectImportArchive = 17,
```

Comment:
nit: Could this be initialized with the value of `BinaryLogRecordKind.ProjectImportArchive` ?